apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.TargetedSessionPool.cs

```diff
@@ -36,7 +36,7 @@ internal sealed class TargetedSessionPool : SessionPoolBase
 
             // Read-only state
             private readonly object _lock = new object();
-            private readonly DatabaseName _databaseName;
+            private readonly SessionPoolSegmentKey _sessionPoolSegmentKey;
```

Comment:
I'd probably just call it _segmentKey, but that doesn't really matter and can be changed later - leave or change as you see fit.

(I'm probably being very inconsistent here, too!)